Library/Formula/libodb-qt.rb

```diff
@@ -0,0 +1,71 @@
+class LibodbQt < Formula
+  homepage "http://www.codesynthesis.com/products/odb/"
+  url "http://www.codesynthesis.com/download/odb/2.4/libodb-qt-2.4.0.tar.gz"
+  sha256 "f48c563f653df178866301e4c5069b09b697d98f54dcc9ef8714157002156183"
+
+  depends_on "libodb" => :recommended
+  option "without-libodb", "Don't install libodb"
+  depends_on "qt" => :optional
+  option "with-qt", "Install the qt library"
+
+  def install
+    qmake = `which qmake`.strip
```

Comment:
Do we really need to shell out here? Homebrew knows if there's a ` qmake ` available. The ` qt ` dep should handle this element automatically; When built ` --with-qt ` it'll make available `qmake` to the formulae.